ensembl_genes/Red.py

```diff
@@ -103,32 +103,31 @@ def fetch_input(self):
                 seq_region[name] = seq_region_id
             self.param('seq_region',seq_region)
         else:
-            raise ValueError('Could not connect to the target database ' \
-                             +target_db_url+' in "target_db_url".')
+            raise ValueError(f'Could not connect to the target database {target_db_url}.')
 
         # make sure that the output directories exist and they are empty
-        if os.path.isdir(msk):
+        if Path(msk).exists():
```

Comment:
The same for an msk_path and an rpt_path.

Reply:
Done. I think I'm liking Path a bit less now since the os option looks more readable and intuitive to me (also using less lines of code). Path also requires keeping an eye on being consistent and not using two implementations for paths, which I think it's not possible to avoid in some cases because Path does not have all methods needed to deal with path operations but let's see how it goes.